sim/death_knight/dancing_rune_weapon.go

```diff
@@ -34,6 +34,14 @@ func (dk *DeathKnight) registerDancingRuneWeaponSpell() {
 
 	hasGlyph := dk.HasMajorGlyph(proto.DeathKnightMajorGlyph_GlyphOfDancingRuneWeapon)
 
+	hasT12Set := dk.HasSetBonus(ItemSetElementiumDeathplateBattlearmor, 4)
+	t124PAura := dk.NewTemporaryStatsAura(
+		"T12 Blood 4P Bonus",
+		core.ActionID{SpellID: 98966},
```

Comment:
The spell id for this one is actually [101162, Flaming Rune Weapon](https://www.wowhead.com/cata/spell=101162/flaming-rune-weapon), just to make it look nicer in the logs instead of "T12 Blood 4P Bonus" :)

Reply:
Nice, done!